mlflow/azureml/__init__.py

```diff
@@ -297,7 +455,7 @@ def _load_pyfunc_conf(model_path):
                 message=("The specified model does not contain the `python_function` flavor. This "
                          " flavor is required for model deployment required for model deployment."),
                 error_code=INVALID_PARAMETER_VALUE)
-    return model.flavors[pyfunc.FLAVOR_NAME]
+    return model.flavors[pyfunc.FLAVOR_NAME], model.run_id
```

Comment:
This change breaks the existing `build_image` implementation. It's also possible that the `run_id` property doesn't exist for the given `Model` object (see https://github.com/mlflow/mlflow/blob/3eaaa046ce3501a48f390b4d778e34e4168aadac/mlflow/models/__init__.py#L41-L43), in which case this method would throw